inlong-dashboard/src/locales/cn.json

```
@@ -167,6 +167,7 @@
   "meta.Sinks.ES.RetryTimes": "重试次数",
   "meta.Sinks.ES.RetryTimesUnit": "次",
   "meta.Sinks.ES.EsVersion": "Elasticsearch 版本",
+  "meta.Sinks.ES.EsVersionHelp": "此处 Elasticsearch 版本为大版本号",
```

Is the following comment correct?
```suggestion
  "meta.Sinks.ES.EsVersionHelp": "填写大版本号，如：对于7.5.2版本，填写7即可",
```

---

like 5,6,7

---

Maybe we can add a drop-down selection box. The options include all supported versions, such as 5/6/7.

---

This is a good way.
However, there will be too much coupling: if the versions supported by the background become more or less, the front end needs to be modified.

---

This limitation is recommended to be placed on the Manager side. It is filled in by the user and verified by the Manager module when saving. If there is any error, the error will be thrown directly.

---

We can deal with it simply and only give a clear reminder.